inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -164,7 +166,9 @@ ie::ICNNNetwork::Ptr FrontEnd::convertNetwork(ie::ICNNNetwork& network) {
         ngraph::op::GenericIE::DisableReshape noReshape(nGraphFunc);
 
         ngraph::pass::Manager manager;
-
+        // WA: ConvertPriorBox must be executed before ConstantFolding pass
+        manager.register_pass<ngraph::pass::InitNodeInfo>();
+        manager.register_pass<ngraph::pass::ConvertPriorBox>();
```

Comment:
Should the same be done in myriad_plugin.cpp and HDDL plugin?

Reply:
As I can see, we don't call Opset1ToLegacy transformations in myriad_plugin.cpp and in HDDL plugin.

Reply:
At the same time, common optimization is called in both myriad_plugin.cpp and frontend.cpp. Do we need to move the call there?

Reply:
@gladilov-gleb ConvertPriorBox is not related to CommonOptimizations, it was a part CommonOptimization due to historical reasons. In an amicable way, this transformation should be included in Opset1ToLegacy, but due to the fact that the transformation must be called before 1st Constant folding, we can't move it that.

Reply:
> the transformation must be called before 1st Constant folding

Before frontend.cpp transformations in myriad_plugin.cpp are applied. Does that mean the even with your changes in frontend.cpp the first Constant Folding is happening before ConvertPriorBox transformation?

Reply:
It seems so.
Why is Common Optimization duplicated (frontend.cpp, myriad.cpp)?

Reply:
At current moment I'm not sure there is still reason for that, so all passes from myriad_plugin.cpp could be moved to frontend.cpp

Reply:
I merged the PR that resolves the issue for you https://github.com/openvinotoolkit/openvino/pull/2220